doc/import-export.rst

```diff
@@ -178,6 +178,34 @@ We will use the `elabapi-python` library to make things easy. See `installation
 
 You can then have a look at `this example to import CSV using the API and metadata/extra fields <https://github.com/elabftw/elabapi-python/blob/master/examples/09-import-csv.py>`_.
 
+.. _compounds-import:
+
+Importing compounds through CLI (csv file)
+------------------------------------------
+
+.. note:: This approach is only available to Sysadmins with shell access.
+
+If you're working with a large database of compounds, using the CLI is a more efficient approach. Display the help with:
+
+.. code-block:: bash
+
+   docker exec -it -u nginx:nginx elabftw bin/console import:compounds -h
```

Comment:
I would drop the -u. First because it might be wrong if the user is not nginx, second because it adds confusion and it's fine to use the default container user root here because it doesn't upload files, just does mysql queries. It's important to have this -u flag when importing things that create files.